assets/templates/test.html

```diff
@@ -0,0 +1,23 @@
+{% load static %}
```

Comment:
Ik gebruik deze pagina tijdelijk om correctheid te controleren in de output (vooral ivm alle annotaties. Let hier niet op, dit is weg ten tijde van de merge

Reply:
True, maar ik heb me nog niet verdiept in jouw test-case structuur en voor potentieel bug fixen had ik zo iets meer feedback, maar dit moet inderdaad een to-do worden